recipe/meta.yaml

```diff
@@ -32,22 +32,25 @@ requirements:
     - python    # [win]
     - cmake >=3.1
     - zlib 1.2*
+    - gcc  # [not win]
+    - libgfortran  # [not win]
```

Comment:
@jakirkham - in https://github.com/conda-forge/hdf5-feedstock/pull/28#discussion_r71445033 (which is more recent that this comment), it sounded like you were saying this line should be kept - do you mean this one should be removed, or the similar line in the runtime dependencies?